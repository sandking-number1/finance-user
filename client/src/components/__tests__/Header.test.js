import React from 'react';
import Header from '../Header.js';
import renderer from 'react-test-renderer';

describe('Header component', () => {
    it('Should match the snapshot', () => {

        const tree = renderer.create(<Header />).toJSON();

        expect(tree).toMatchSnapshot();
    });
});