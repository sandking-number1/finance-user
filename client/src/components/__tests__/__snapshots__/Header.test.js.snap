// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component Should match the snapshot 1`] = `
<header>
  <div
    class="row banner"
  >
    <div
      class="col-8"
    >
      <h1>
        R.I.S.E.
      </h1>
    </div>
    <div
      class="col-4"
    >
      <h3>
        POP POS
      </h3>
    </div>
  </div>
</header>
`;
